miqa/celery.py

```diff
@@ -14,3 +16,11 @@
 
 # Load task modules from all registered Django app configs.
 app.autodiscover_tasks()
+
+if settings.DEMO_MODE:
+    app.conf.beat_schedule = {
+        'reset-demo': {
+            'task': 'miqa.core.tasks.reset_demo',
+            'schedule': timedelta(days=1),
```

Comment:
You might want more control of the exact time at which it executes, rather than just specifying the interval between executions. Consider using a [Celery crontab schedule](https://docs.celeryq.dev/en/stable/userguide/periodic-tasks.html#crontab-schedules) like:
```suggestion
            'schedule': crontab(minute=0, hour=0),  # daily at midnight
```

You'll also need:
```python
from celery.schedules import crontab
```

Reply:
Changed in [5a36a70](https://github.com/OpenImaging/miqa/pull/635/commits/5a36a70f895942267b886b82c0ed7cabb62d4a76)